knowledge-base/treelist-enable-checkbox-selection-only-edit-mode.md

```diff
@@ -0,0 +1,354 @@
+---
```

Comment:
The customer user the word "disable". In our KB, we use the word "enable" more often than "disable". No need to change this, but just saying that we should try to preserve the customer context and keywords when they are technically correct.